contrib/firebaseio/src/main/java/contrib/JacksonCoder.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not  use this file except  in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package contrib;
```

Comment:
the package names seem to have gotten a little wonky. I cannot tell exactly when.

This should be `com.google.cloud.dataflow.contrib.firebase`, and the same throughout. (See, e.g., https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/contrib/hadoop/src/main/java/com/google/cloud/dataflow/contrib/hadoop/HadoopFileSource.java#L17)

[Feel free to use subpackages of c.g.c.d.contrib.firebase within the module as you see fit, of course]